### PR TITLE
J-002: use explicit `None` check

### PR DESCRIPTION
**Pull request by wthlolbbq**

### CODE REVIEW

File: bot/cmd/calculator.py
Comment (Alty-dev) [inline bot/cmd/calculator.py:1]: very nice

**Review by Alty-dev — COMMENTED** [review]
very nice

File: bot/cmd/calculator.py
Comment (wthlolbbq) [reply bot/cmd/calculator.py:1]: Thanks.

**Comment by wthlolbbq** [comment]
#2

**Comment by wthlolbbq** [comment]
closes #2

**Pull request merged**